nsv13/code/modules/munitions/ship_weapons/hybrid_weapons/hybrid_railgun.dm

```diff
@@ -116,7 +116,7 @@
 			after_fire()
 	return FALSE
 
-/obj/machinery/ship_weapon/hybrid_rail/can_fire(shots = weapon_type.burst_size)
+/obj/machinery/ship_weapon/hybrid_rail/can_fire(target, shots = weapon_type.burst_size) //Target is for the passed target variable, Shots is for the burst fire size
```

Comment:
This should imo be explained in a doc comment on machinery/ship weapon/proc/can_fire, not here. I can't see the use this comment has for future readers.